ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -270,8 +270,16 @@ def get_action(
         )
 
         self.save_memories(global_agent_ids, run_out.get("memory_out"))
+        action = run_out.get("action")
+        # Fast NaN check on the action
+        # See https://stackoverflow.com/questions/6736590/fast-check-for-nan-in-numpy for background.
+        d = np.sum(action)
```

Comment:
There's a few ways to do this; we were using `np.dot` for observations at one point:
https://github.com/Unity-Technologies/ml-agents/blob/81acfaf1baa93ffd477b1b25e9e668c705a1a56a/ml-agents-envs/mlagents/envs/brain.py#L245
but that morphed to `np.mean` during some refactoring:
https://github.com/Unity-Technologies/ml-agents/pull/3022/files#diff-614d11115c4d74319295ea34b44a3b7d8ab40b43865c1c3e351a8f6c9602c673R92

`np.sum` is one of the recommended approaches in the stackoverflow thread.